packaging/generic-deb-integrated/debian/README.source

```diff
@@ -14,7 +14,7 @@ Steps to build and install package:
     https://docs.aws.amazon.com/efs/latest/ug/installing-amazon-efs-utils.html#installing-other-distro
 
 5. Install the package with
-    sudo apt install -y ./amazon-ecs-init_1.45.0-1_amd64.deb
+    sudo apt install -y ./amazon-ecs-init_1.66.2-1_amd64.deb
```

Comment:
Is there any reason we can't use the ${VERSION} variable here?

Reply:
Yes, we can do that. I can possibly leave it at ```./amazon-ecs-init_${AGENT_VERSION}-1_amd64.deb``` or ```./amazon-ecs-init_${INIT_VERSION}_amd64.deb``` to make it look more generic. User would need to update this while running in both cases